### PR TITLE
OSD-15584: Show only a single view for trigerred incidents

### PR DESCRIPTION
**Pull request by Deepanshu276**
### What type of PR is this?

cleanup

### What this PR does / Why we need it?
Initially there was two views for listing triggered incidents; one view lists the incidents and the other lists the alerts,which was not ideal

This PR changes the following

- Removes the triggered alerts view from kite
- Upon clicking on the triggered incident, kite list alerts related to that incident

Screenshot from 2023-03-22 17-38-12

Screenshot from 2023-03-22 17-38-17

### Which Jira/Github issue(s) does this PR fix?

_Resolves [[OSD-15584](https://issues.redhat.com/browse/OSD-15584)] : Show only a single view for trigerred incidents

### Special notes for your reviewer

NA
### Pre-checks (if applicable)

- [x] Ran unit tests locally against the changes
- [x] Included documentation changes with PR

### CODE REVIEW

**Comment by supreeth7** [comment]
@Deepanshu276 can you please rebase?

**Comment by Deepanshu276** [comment]
Yeah sure

File: pkg/ui/constants.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ const (
 	// Page Titles
 	AlertsPageTitle          = "Alerts"
 	AlertDataPageTitle       = "Metadata"
+	AlertMetadata            = "AlertData"
```
Comment (supreeth7) [inline pkg/ui/constants.go:25]: Why is this needed?

**Review by supreeth7 — CHANGES_REQUESTED** [review]
Before I review further, I've also noticed that pressing the keys `1` or `2` don't switch between the relevant views. Can this be looked at?

**Comment by Deepanshu276** [comment]
When we press keys 1 and 2, it don't get switched to relevant views as when we integrated kite with tmux , it has some switching tabs issue. It overrides because of some reason and we will resolve it soon.

File: pkg/ui/constants.go (hunk 1 above)
Comment (Deepanshu276) [reply pkg/ui/constants.go:25]: When we view the sop data and if we want to return back to triggered incident view , then this helps us to move back from metadata view to back to triggered incident view.

**Review by supreeth7 — APPROVED** [review]
/lgtm
/approve
